### ISSUE TITLE
Bubble Charts limited to 26 columns 

### ISSUE THREAD

**Issue by benjaminpavone** [gitbrent/PptxGenJS#1076]
### Category
Bug

### Version
Please specify what version of the library you are using: [  3.9   ]

### Expected / Desired Behavior
Hi
Bubble charts are currently limited to columns named in range [A-Z]. With more than 26 columns the generated xml has an error.

File gen-charts.ts : 
```
if (chartObject.opts._type === CHART_TYPE.BUBBLE) {
    strSheetXml += '<dimension ref="A1:' + LETTERS[intBubbleCols - 1] + (data[0].values.length + 1) + '" />'
}
```
### Steps to reproduce

A sample code here : https://jsfiddle.net/7gc2kp0z/5/
With more than 26 columns, when you tried to change series in the generated pptx, you get an error. (export is good, bubbles too, but you can't change series)

After looking in generated xml files : 
dimension ref="A1:undefined17" should be dimension ref="A1:AG17" (for example)

Best regards

**Comment by thiagoflessak** [comment]
Actually, I think this issue will happen to any chart with more than 26 series. I've faced this problem with a bar chart.

**Comment by gitbrent** [comment]
### Bug Details
Several methods within `gen-charts.ts` are using `LETTERS[]` instead of the `getExcelColName()` function to derive column names.

**Comment by gitbrent** [comment]
Thanks for reporting this @benjaminpavone 

Fix will be in v3.11.0
